ElasticBox/user-authentication.md

```diff
@@ -7,12 +7,30 @@
 }}}
 
 ### Enable User Authentication
+<<<<<<< HEAD
```

Comment:
Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.